addons/knowledge/security/ir.model.access.csv

```diff
@@ -0,0 +1,16 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_knowledge_article_all,access.knowledge.article.all,knowledge.model_knowledge_article,,0,0,0,0
```

Comment:
unnecessary `,,0,0,0,0` rules, it's implied that you don't have access if not mentionned

Reply:
we prefer to be explicit, but indeed.